bench/wtperf/runners/wtperf_run.sh

```diff
@@ -17,27 +17,27 @@ if test "$#" -lt "2"; then
 	exit 1
 fi
 wttest=$1
-runmax=$2
+shift # Consume this arg
+runmax=$1
+shift # Consume this arg
 # Jenkins removes the quotes from the passed in arg so we may
 # have 3 or 4 args.
```

Comment:
Does this comment still apply, now that script would allow arbitrary number of arguments.